environments.yaml.tmpl

```diff
@@ -1,10 +1,24 @@
 environments:
- default:
-   values:
-   - ../etc/base.yaml
-   - ../etc/production.yaml
-   - ../etc/production.yaml.gotmpl
-   - ../etc/secrets.yaml
+  default:
+    values:
+      - ../etc/base.yaml
+      - ../etc/production.yaml
+      - ../etc/production.yaml.gotmpl
+      - ../etc/secrets.yaml
+    {{ if not .Values.enable_tls }}
+      - ../mods/disable_tls.yaml
```

Comment:
To reduce confusion I think these 2 lines should either be `enable_tls` or `disable_tls`

Reply:
It is a matter of API design, I guess. The config file disables tls and is, therefore, correctly named. What are the config options that we would like to present:

A:
```properties
enable_logging_monitoring: false
# Enable TLS redirection and retrieval of Let's Encrypt certificates.
# Can be disabled when TLS termination is handled upstream of the on-cluster Nginx reverse proxy.
enable_tls: true
```

B:
```properties
disable_logging_monitoring: true
# Enable TLS redirection and retrieval of Let's Encrypt certificates.
# Can be disabled when TLS termination is handled upstream of the on-cluster Nginx reverse proxy.
disable_tls: false
```

I am fine either way.

Reply:
If there is a `/mods/disable_tls.yaml` and I think the variable should also be `disable_tls`